tests/ctst/common/common.ts

```diff
@@ -353,8 +353,7 @@ When('the user tries to perform the current S3 action on the bucket {int} times
             }
             await runActionAgainstBucket(this, this.getSaved<ActionPermissionsType>('currentAction').action);
             if (this.getResult().err) {
```

Comment:
maybe better to keep failing on error, except for "retriable" errors : which can be safey ignore as this step is indeed about making multiple attempts

Reply:
I updated CTST and now we fail if the error is not retryable